File: src/main/java/block/BlockOre.java
package block;

import registry.RegistryName;
import world.World;

public class BlockOre extends Block {
    public BlockOre(RegistryName name) {
        super(name);
    }

    @Override
    public boolean shouldRenderFace(World world, int cX, int cZ, int rY, int x, int y, int z, EnumDirection face) {
        return super.shouldRenderFace(world, cX, cZ, rY, x, y, z, face);
    }
}

File: src/main/kotlin/block/BlockOre.kt
package block

import registry.RegistryName
import world.World

class BlockOre(_name: RegistryName) : Block(_name) {

    override fun shouldRenderFace(
        world: World,
        cX: Int,
        cZ: Int,
        rY: Int,
        x: Int,
        y: Int,
        z: Int,
        face: EnumDirection
    ): Boolean {
        return super.shouldRenderFace(world, cX, cZ, rY, x, y, z, face)
    }
}